dips/dip-1.mdx

```diff
@@ -332,7 +332,7 @@ A `PaymentActorObject` represents a participant in a payment - either sender or
 
 | Field 	    | Type 	| Required? 	| Description 	|
 |-------	    |------	|-----------	|-------------	|
-| address | str | Y | Address of the sender/receiver account. Addresses may be single use or valid for a limited time, and therefore VASPs should not rely on them remaining stable across time or different VASP addresses. The addresses are encoded using bech32. The bech32 address encodes both the address of the VASP as well as the specific user's subaddress. They should be no longer than 80 characters. Mandatory and immutable. For Diem addresses, refer to the "account identifier" section in DIP-5 for format. |
+| address | str | Y | The 16-byte on-chain address of the sender/receiver account in case-insensitive, hex encoding. |
```

Comment:
I'm not sure about hex encoding or bech32 encoding here. Everywhere else in this dip uses bech32. Better to make them consistent.

Reply:
lemme know if you see any other bech32 in here, because we don't have a bech32 encoder / decoder for just an account address

Reply:
One question about this change is the HRP of the account identifier was used for checking the network of the payment. The account address may belongs to multiple network, this maybe confusing to sender/receiver what network sender is going to submit the transaction.